lib/node_modules/@stdlib/repl/lib/filter_by_prefix.js

```diff
@@ -21,6 +21,73 @@
 // MODULES //
 
 var startsWith = require( '@stdlib/string/starts-with' );
+var max = require( '@stdlib/math/base/special/max' );
+var abs = require( '@stdlib/math/base/special/abs' );
+
+
+// FUNCTIONS //
+
+/**
+* Checks if the completion is a fuzzy match for the input.
+*
+* A fuzzy match is determined by the number and order of matching characters, penalizing large distances between matches.
+* A score above or equal to 0.8 indicates a match.
```

Comment:
I think these notes may be better localized below as inline comments. Less likely to experience implementation-note drift, should the implementation ever be refactored.

Reply:
Also, it seems as though the score note does not match the implementation.